zap/src/main/java/org/zaproxy/zap/extension/api/CoreAPI.java

```diff
@@ -1294,6 +1307,32 @@ public void process(RecordHistory recordHistory) {
             result =
                     new ApiResponseElement(
                             name, getAlertParam(ApiException.Type.BAD_VIEW).getOverridesFilename());
+        } else if (VIEW_LOGGING_GENERAL.equals(name)) {
+            var config = LoggerContext.getContext().getConfiguration();
+            result =
+                    new ApiResponseSet<String>(
+                            name,
+                            Map.of(
+                                    PAROS_PACKAGE,
+                                    config.getLoggerConfig(PAROS_PACKAGE).getLevel().name(),
+                                    ZAP_PACKAGE,
+                                    config.getLoggerConfig(ZAP_PACKAGE).getLevel().name()));
+        } else if (VIEW_LOGGING_DETAILED.equals(name)) {
+            String basePackage = ApiUtils.getOptionalStringParam(params, PARAM_BASE_PACKAGE);
+            SortedMap<String, String> loggerDetails = new TreeMap<>();
+            for (Logger logger : LoggerContext.getContext().getLoggers()) {
+                if (!logger.getName().contains(".")) {
```

Comment:
I don't think we should do this since the base levels matter too and not all names used in the codebase have dot (e.g. https://github.com/zaproxy/community-scripts/blob/a941beb79d9576c684bb1724b146b3c81b7d5fd7/active/SSTI.js#L12).

Reply:
Oh okay, I wasn't aware of that.